systemtest/src/main/java/io/strimzi/systemtest/utils/specific/BridgeUtils.java

```diff
@@ -4,192 +4,76 @@
-                            String kafkaTopic = jsonResponse.getString("topic");
-                            int kafkaPartition = jsonResponse.getInteger("partition");
-                            String key = jsonResponse.getString("key");
-                            Object value = jsonResponse.getValue("value");
-                            long offset = jsonResponse.getLong("offset");
-                            LOGGER.debug("Received msg: topic:{} partition:{} key:{} value:{} offset{}", kafkaTopic, kafkaPartition, key, value, offset);
-                        }
-                        LOGGER.info("Received {} messages from KafkaBridge", response.body().size());
-                    } else {
-                        LOGGER.warn("Received body 0 messages: {}", response.body());
-                    }
-                    future.complete(response.body());
-                } else {
-                    LOGGER.info("Cannot consume any messages!", ar.cause());
-                    future.completeExceptionally(ar.cause());
-                }
-            });
-        return future.get(1, TimeUnit.MINUTES);
+    public static String executeCurlCommand(HttpMethod httpMethod, String podName, String url, String headers) {
+        return executeCurlCommand(httpMethod, podName, "", url, headers);
     }
 
-    public static boolean subscribeHttpConsumer(JsonObject topics, String bridgeHost, int bridgePort, String groupId,
-                                                String name, WebClient client, Map<String, String> additionalHeaders) throws InterruptedException, ExecutionException, TimeoutException {
-
-        MultiMap headers = MultiMap.caseInsensitiveMultiMap()
-            .add("Content-length", String.valueOf(topics.toBuffer().length()))
-            .add("Content-type", Constants.KAFKA_BRIDGE_JSON);
-
-        for (Map.Entry<String, String> header : additionalHeaders.entrySet()) {
-            LOGGER.info("Adding header {} -> {}", header.getKey(), header.getValue());
-            headers.add(header.getKey(), header.getValue());
-        }
-
-        CompletableFuture<Boolean> future = new CompletableFuture<>();
-
-        client.post(bridgePort, bridgeHost,  "/consumers/" + groupId + "/instances/" + name + "/subscription")
-            .putHeaders(headers)
-            .as(BodyCodec.jsonObject())
-            .sendJsonObject(topics, ar -> {
-                LOGGER.info(ar.result());
-
-                if (ar.succeeded() && ar.result().statusCode() == 204) {
-                    LOGGER.info("Consumer subscribed");
-                    future.complete(ar.succeeded());
-                } else {
-                    LOGGER.error("Cannot subscribe consumer", ar.cause());
-                    future.completeExceptionally(ar.cause());
-                }
-            });
-        return future.get(1, TimeUnit.MINUTES);
+    public static String executeCurlCommand(HttpMethod httpMethod, String podName, String data, String url, String headers) {
```

Comment:
I know that the base idea was good. In a practise if you create method, which has one line to encapsulate some logic. You can end-up having these `one-liners` with different names but most likely same behaviour. Wouldn't you think that having:
```
cmdKubeClient().execInPod(podName, "/bin/bash", "-c", BridgeUtils.getCurlCommand(httpMethod, url, headers, data)).out().trim();
```
will be better?